tests/dummy/app/components/smiley/template.hbs

```diff
@@ -0,0 +1,5 @@
+<@parent.Circle @x={{@x}} @y={{@y}} @radius={{@radius}} @fill={{@back}} @stroke={{@fore}} @strokeWidth={{2}} @draggable={{true}} />
+<@parent.Circle @x={{leftEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
+<@parent.Circle @x={{rightEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
+<@parent.Arc @x={{@x}} @y={{mouthY}} @rotation={{70}} @innerRadius={{40}} @outerRadius={{43}} @angle={{40}} @fill={{@fore}} />
+<@parent.Text @text={{@name}} @x={{nameX}} @width={{nameWidth}} @y={{nameY}} @align="center" /> 
```

Comment:
To make the entire thing draggable you would need to have a single Konva node to group them together, therefor a `Group`.

Suggestion:
```suggestion
<@parent.Group @draggable={{true}} as |G|>
  <G.Circle @x={{@x}} @y={{@y}} @radius={{@radius}} @fill={{@back}} @stroke={{@fore}} @strokeWidth={{2}} />
  <G.Circle @x={{leftEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
  <G.Circle @x={{rightEyeX}} @y={{eyeY}} @radius={{4}} @fill={{@fore}} />
  <G.Arc @x={{@x}} @y={{mouthY}} @rotation={{70}} @innerRadius={{40}} @outerRadius={{43}} @angle={{40}} @fill={{@fore}} />
  <G.Text @text={{@name}} @x={{nameX}} @width={{nameWidth}} @y={{nameY}} @align="center" />
</@parent.Group>
```

Note the `@draggable` is moved from the first circle and onto the group